AppKit/CPSplitView.j

```diff
@@ -1205,6 +1186,29 @@ The sum of the views and the sum of the dividers should be equal to the size of
 
 @end
 
+@implementation CPSplitView (CPTrackingArea)
+
+- (void)updateTrackingAreas
+{
+    [self removeAllTrackingAreas];
+
+    var options = CPTrackingCursorUpdate | CPTrackingActiveInKeyWindow;
```

Comment:
Right, because of the loop, I would have done the same. No need to change the others.